package.json

```diff
@@ -181,60 +181,60 @@
     "babili-webpack-plugin": "0.1.2",
     "chai": "4.1.2",
     "concurrently": "3.5.1",
-    "cross-env": "5.1.4",
+    "cross-env": "5.1.6",
     "css-loader": "0.28.11",
     "devtron": "1.4.0",
     "difflet": "1.0.1",
-    "drivelist": "6.1.8",
+    "drivelist": "6.2.2",
     "electron": "1.8.4",
-    "electron-builder": "20.11.1",
-    "electron-devtools-installer": "2.2.3",
-    "electron-mocha": "6.0.2",
+    "electron-builder": "20.15.1",
+    "electron-devtools-installer": "2.2.4",
+    "electron-mocha": "6.0.4",
     "electron-updater": "2.21.10",
     "enzyme": "3.3.0",
     "eslint": "4.19.1",
     "eslint-config-airbnb": "16.1.0",
     "eslint-formatter-pretty": "1.3.0",
-    "eslint-import-resolver-webpack": "0.9.0",
-    "eslint-plugin-import": "2.11.0",
+    "eslint-import-resolver-webpack": "0.10.0",
+    "eslint-plugin-import": "2.12.0",
     "eslint-plugin-jsx-a11y": "6.0.3",
     "eslint-plugin-lodash": "2.7.0",
     "eslint-plugin-mocha": "5.0.0",
-    "eslint-plugin-promise": "3.7.0",
-    "eslint-plugin-react": "7.7.0",
+    "eslint-plugin-promise": "3.8.0",
+    "eslint-plugin-react": "7.9.1",
     "express": "4.16.3",
     "extract-text-webpack-plugin": "3.0.2",
     "fbjs-scripts": "0.8.3",
     "file-loader": "1.1.11",
-    "flux-standard-action": "2.0.2",
+    "flux-standard-action": "2.0.3",
     "ftp": "0.3.10",
     "git-describe": "4.0.2",
     "gitbook-cli": "2.3.2",
     "html-webpack-plugin": "3.2.0",
     "json-loader": "0.5.7",
-    "less": "3.0.2",
+    "less": "3.0.4",
     "less-loader": "4.1.0",
     "minimist": "1.2.0",
-    "mocha": "5.1.1",
+    "mocha": "5.2.0",
     "node-hid": "0.7.2",
-    "nodegit": "0.22.0",
+    "nodegit": "0.22.1",
     "object-invariant-test-helper": "0.1.1",
     "open": "0.0.5",
-    "react-hot-loader": "4.1.2",
+    "react-hot-loader": "4.3.2",
     "redux-mock-store": "1.5.1",
     "salinity": "0.0.8",
     "serialport": "6.2.0",
-    "shelljs": "0.8.1",
-    "sinon": "5.0.3",
-    "sinon-chai": "3.0.0",
+    "shelljs": "0.8.2",
+    "sinon": "6.0.0",
```

Comment:
There shouldn't be any breaking changes between v5 and v6: http://sinonjs.org/guides/migrating-to-6.0